File: Selenium/Activities/selenium_activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By

#create WebDriver instance
with webdriver.Firefox() as driver:

    #go to web page
    driver.get("https://training-support.net/webelements/dynamic-controls")

    #do the stuff on the page
    pageTitle = driver.title
    print("Title of the page is : " + pageTitle)

    #Find the 3rd header and print its text
    checkbox = driver.find_element(By.ID, "checkbox")
    print("Checkbox is visible: " , checkbox.is_displayed())

    #Find the 5th header and print its color
    print("Click the Toggle Checkbox button - ")
    checkbox_toggle = driver.find_element(By.XPATH, "/html/body/div/main/div/div/div/div/div[2]/section[1]/button").click()
    	
	#Find the violet button and print its classes
    print("Checkbox is visible: " , checkbox.is_displayed())
    
    #close the browser
    driver.quit()

File: Selenium/Activities/selenium_Activity5.java
package selenium_session1;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class selenium_Activity5 
{
	public static void main(String[] arg)
	{
		//create a WebDriver instance
		WebDriver driver = new FirefoxDriver();
				
		//go to a web page
		driver.get("https://training-support.net/webelements/dynamic-controls");
		
		//do stuff on the page
		System.out.println("Title of the page is : " + driver.getTitle());
		
		WebElement element = driver.findElement(By.name("checkbox"));
		System.out.println("Checkbox is visible: " + element.isDisplayed());
		
		System.out.println("Click the Toggle Checkbox button - ");
		driver.findElement(By.xpath("//button[contains(@class, 'svelte-sfj3o4')]")).click();
		//driver.findElement(By.xpath("/html/body/div/main/div/div/div/div/div[2]/section[1]/button")).click();
		
		System.out.println("Checkbox is visible: " + element.isDisplayed());
		
		//close the browser
		driver.quit();
	}

}
